packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -189,6 +191,47 @@ export class PlaygroundWorkerEndpoint extends PHPWorker {
 	}
 }
 
+async function downloadWordPressAssets(php: PHP) {
+	const wpVersion = await getWordPressVersionFromPhp(php);
+	fetch(`${wordPressSiteUrl}/wp-${wpVersion}/wordpress-static.zip`)
```

Comment:
When the `wordpress-static.zip` is missing and the response status is 404, there is no warning logged to the console. Errors are caught within the anonymous async function below, but there is no top-level error/rejection callback to warn for the 404.

Reply:
This was found in local testing with a WP version built without wordpress-static.zip.

Reply:
I added a warning in case the download isn't ok [3c98912](https://github.com/WordPress/wordpress-playground/pull/1532/commits/3c98912b1dca7c621fe8ac081fbe34d8bc8a6c0f)